src/functionalTest/resources/requests/roleAssignment/set-organisational-role-assignment-request.json

```diff
@@ -6,7 +6,7 @@
       "classification": "PUBLIC",
       "grantType": "STANDARD",
       "readOnly": false,
-      "roleCategory": "STAFF",
+      "roleCategory": "LEGAL_OPERATIONS",
```

Comment:
this is used when creating a role-assignment has the request role category change completely or can we still use STAFF

Reply:
STAFF is removed and replaced with LEGAL_OPERATIONS